internal/policy/container/max_layers.go

```diff
@@ -25,15 +25,15 @@ func (p *MaxLayersCheck) Validate(ctx context.Context, imgRef certification.Imag
 		return false, fmt.Errorf("could not get image layers: %v", err)
 	}
 
-	return p.validate(layers)
+	return p.validate(ctx, layers)
 }
 
 func (p *MaxLayersCheck) getDataToValidate(image cranev1.Image) ([]cranev1.Layer, error) {
 	return image.Layers()
 }
 
-func (p *MaxLayersCheck) validate(layers []cranev1.Layer) (bool, error) {
-	log.L().Debugf("detected %d layers in image", len(layers))
+func (p *MaxLayersCheck) validate(ctx context.Context, layers []cranev1.Layer) (bool, error) {
+	logr.FromContextOrDiscard(ctx).V(5).Info("number of layers detected in image", "layerCount", len(layers))
```

Comment:
This needs to change to be `V(1)`?

Reply:
I marked this in other places where I also saw `V(5)`, but the fact that it exists in multiple places has me unsure that we actually want to swap it.

Reply:
That was missed since I did a replace all.. And it didn't match my search term.